cv_bridge/CMakeLists.txt

```diff
@@ -12,7 +12,13 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   add_compile_options(-Wall -Wextra)
 endif()
 
+option(CV_BRIDGE_DISABLE_PYTHON "Build Python package" OFF)
+
 if(ANDROID)
+  set(CV_BRIDGE_DISABLE_PYTHON ON)
+endif()
+
+if(CV_BRIDGE_DISABLE_PYTHON)
   find_package(Boost REQUIRED)
```

Comment:
I see that ``find_package(Boost REQUIRED)`` still runs even if you disable cv bridge. Is this problematic for your platform or is it fine?

Reply:
Boost itself is fine as only `boost_python` and `boost_coroutine` which requires some assembly code are missing.

Reply:
Got it, thanks for the clarification!